app/controllers/application_controller.rb

```diff
@@ -157,6 +158,9 @@ def download_summary_pdf
     @embedded = true
     @showlinks = false
 
+    # encode images and embed in HTML that is sent to Prince
+    @base64_encode_images = true
```

Comment:
Upstream code does not have Prince by default, though I think the only other choice is the NullPdf plugin.  Even so, if/when we ever create a new PDF plugin, will this base64 encoding of images still be needed there?  If not, should this be a property of the PDF plugin system?